rest/clients/jaxrs-cxf/pom.xml

```diff
@@ -0,0 +1,121 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Copyright (c) 2019 Connexta, LLC
+
+    Released under the GNU Lesser General Public License version 3; see
+    https://www.gnu.org/licenses/lgpl-3.0.html
+
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>com.connexta.search</groupId>
+    <artifactId>query-api-rest-clients</artifactId>
+    <version>0.4.0-SNAPSHOT</version>
+  </parent>
+
+  <name>Query API :: REST JAX-RS CXF Client Stubs</name>
+
+  <artifactId>query-api-rest-jaxrscxf-client-stubs</artifactId>
+
+  <dependencies>
+    <!--<dependency>-->
+    <!--  <groupId>com.connexta.search</groupId>-->
+    <!--  <artifactId>query-api-rest-specs</artifactId>-->
+    <!--  <version>${project.version}</version>-->
+    <!--  <type>zip</type>-->
+    <!--</dependency>-->
+    <dependency>
+      <groupId>com.connexta.search</groupId>
+      <artifactId>query-api-rest-models</artifactId>
+      <version>${project.version}</version>
+    </dependency>
+    <dependency>
+      <groupId>javax.validation</groupId>
+      <artifactId>validation-api</artifactId>
+    </dependency>
+    <!--
+        Replace io.springfox:springfox-swagger2 with io.swagger.core.v3:swagger-core when
+        the generator supports io.swagger.v3.annotations.
+    -->
+    <dependency>
+      <groupId>io.springfox</groupId>
+      <artifactId>springfox-swagger2</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>javax.ws.rs</groupId>
+      <artifactId>javax.ws.rs-api</artifactId>
+      <!-- This is the version in DDF. -->
+      <version>2.1</version>
+    </dependency>
+  </dependencies>
+
+  <build>
+    <plugins>
+      <!--<plugin>-->
```

Comment:
✏️ Delete.

Reply:
I prefer keeping these lines so that I can easily generate the classes if I need to since we try to keep our classes as close to possible as the generated ones.

Reply:
Maybe this could be done in a profile.